otx/algorithms/classification/adapters/dor/__init__.py

```diff
@@ -0,0 +1,15 @@
+"""Adapters of classification - dor (deep-object-reid)."""
```

Comment:
How about using 'deep-object-reid' instead of 'dor' ?
When we handle the other public repository, I think it would be better that note the original name

Reply:
Deep-object-reid may not be appropriate for modularity. How about `deep_reid` or `deep_object_reid?`

Reply:
I revised it to deep_object_reid